pom.xml

```diff
@@ -16,7 +16,8 @@
   </modules>
   
   <properties>
-    <version.org.jpmml.evaluator>1.4.9</version.org.jpmml.evaluator> <!-- ATTENTION 1.4.11 intentional, because 1.4.9 evaluators depend on 1.4.11 --> 
+    <version.org.jpmml.evaluator>1.4.9</version.org.jpmml.evaluator> <!-- ATTENTION 1.4.11 intentional, because 1.4.9 evaluators depend on 1.4.11 -->
+    <version.org.glassfish.jaxb>2.4.0-b180830.0438</version.org.glassfish.jaxb>
```

Comment:
@danielezonca just curious - how did you pick exactly this version?

Reply:
Just because it is the last one I found on Maven Central :smile: 
https://mvnrepository.com/artifact/org.glassfish.jaxb/jaxb-runtime/2.4.0-b180830.0438
FYI it works with version 2.4.0-b180725.0644 too

Reply:
Aha ok cool +1